.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "2.7"
+  - "3.6"
```

Comment:
You also need to update the commands at https://github.com/anjali-dhanuka/vms/blob/8ebafec862a960efcdd4d612a97116f1dfabe35c/.travis.yml#L17, `syncdb` will not work.

Reply:
@tapasweni-pathak Commands updated.
Please have a look now.